.github/workflows/documentation.yml

```
@@ -0,0 +1,41 @@
+name: Documentation
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        lfs: true
+    - name: Install Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'    
```

I would use a specific version here. It's possible that when 3.11 is released in a couple weeks sphinx is initially incompatible. I doubt that will be the case, but probably better to stick with a known working version.

```suggestion
        python-version: '3.10'
```

---

changed to specific version